vaadin-upload-flow-parent/vaadin-upload-flow/src/main/java/com/vaadin/flow/component/upload/receivers/FileBuffer.java

```diff
@@ -26,6 +26,7 @@
 import java.util.logging.Level;
 
 import com.vaadin.flow.component.upload.Receiver;
+import com.vaadin.flow.server.streams.FileFactory;
```

Comment:
This looks like a breaking change? This wouldn't compile anymore:
```java
com.vaadin.flow.component.upload.receivers.FileFactory factory = new com.vaadin.flow.component.upload.receivers.FileFactory() {
    @Override
    public java.io.File createFile(String fileName) throws IOException {
        return java.nio.file.Files.createTempFile(fileName, null).toFile();
    }
};

FileBuffer fileBuffer = new FileBuffer(factory);
```

Also I don't see why this would need to be changed at this point. Or ever, because all the file factory stuff should probably be in Flow, or already is with classes like `TemporaryFileUploadHandler`.

Reply:
reverted